packages/h3/src/H3Adapter.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  AppControllerRoute,
+  AppViewRoute,
+  BullBoardQueues,
+  ControllerHandlerReturnType,
+  IServerAdapter,
+  UIConfig,
+} from "@bull-board/api/dist/typings/app";
+import fs from "fs";
+import { createRouter, eventHandler, getRouterParams, setResponseHeader, getQuery } from "h3";
+import ejs from "ejs";
+
+export class H3Adapter implements IServerAdapter {
+  private uiHandler = createRouter();
+  private basePath = "/ui";
+  private bullBoardQueues: BullBoardQueues | undefined;
+  private viewPath: string | undefined;
+  private uiConfig: UIConfig = {};
+
+  public setBasePath(path: string): H3Adapter {
+    this.basePath = path;
+    return this;
+  }
+
+  private getStaticFile(path: string, filename: string) {
+    return fs.readFileSync(`${this.viewPath}${path}/${filename}`, "utf-8");
+  }
+
+  private getContentType(filename: string) {
+    let contentType = "text/html";
+
+    switch (filename.split(".").pop()) {
+      case "js":
+        contentType = "text/javascript";
+        break;
+      case "css":
+        contentType = "text/css";
+        break;
+      case "png":
+        contentType = "image/png";
+        break;
+      case "svg":
+        contentType = "image/svg+xml";
+        break;
+      case "json":
+        contentType = "application/json";
+        break;
+      case "ico":
+        contentType = "image/x-icon";
+        break;
+    }
+
+    return contentType;
+  }
+
+  public setStaticPath(staticsRoute: string, _staticsPath: string): H3Adapter {
+    this.uiHandler
+      .get(
+        this.basePath,
+        eventHandler(async () => {
+          return ejs.renderFile(this.viewPath + "/index.ejs", {
+            basePath: `${this.basePath}/`,
+            title: this.uiConfig.boardTitle || "BullMQ",
```

Comment:
I think `??` is preferred over `||`.

Reply:
Why do you need this title field? 
I doesn't exist in other adapters (uiConfig is missing here)